src/CTA.Rules.Metrics/Models/DownloadedFilesMetric.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace CTA.Rules.Metrics
+{
+    public class DownloadedFilesMetric : CTAMetric
+    {
+        [JsonProperty("metricName", Order = 10)]
+        public string MetricName => "RulesFile";
+
+        [JsonProperty("downloadedFile", Order = 11)]
+        public string DownloadedFile { get; set; }
+
+        [JsonProperty("solutionPath", Order = 12)]
```

Comment:
do you think "solutionPath" / "projectguid" etc (used more than once), could be moved to a constant file ?

Reply:
Generally, I agree that string literals should be stored in a constants file, but these strings are used to rename these properties when the instances of the class are serialized to json (we do this when we are exporting the metrics to logs/files). Since the output json is decoupled from the application itself, I don't think there would be much reusability in storing these string values as constants.

Reply:
the point is when you are changing "solutionPath" to "SolutionFullPath" across all metrics.. you might have to change across many CS files instead of one.... so evaluate and make changes if needed

Reply:
Oh, I understand now. Making the property name uniform across multiple metrics would warrant a change here. Thank you for pointing that out. I can open an issue and create a separate PR for that.